### PR TITLE
[DOCS] Add Take action menu to Alert details flyout in Timeline

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Fixes https://github.com/elastic/security-docs/issues/1853.

Preview [here](https://security-docs_1854.docs-preview.app.elstc.co/guide/en/security/master/timelines-ui.html#timeline-inspect-events-alerts).

### CODE REVIEW

**Review by joepeeples — COMMENTED** [review]
Writing looks fine, but I think this might need to be more general than just alert details. Timeline can also display any event, even if it's not a detection alert, and a similar details flyout and "Take action" button appears for non-alert events, too.

**Review by joepeeples — APPROVED** [review]
LGTM!

**Review by michaelolo24 — APPROVED** [review]
Thanks for this change! LGTM!

**Review by benironside — APPROVED** [review]
✅  👍

File: docs/events/timeline-ui-overview.asciidoc [hunk 1]
```diff
@@ -32,6 +32,11 @@ retrieved from the alert. For more information, refer to <<timeline-templates-ui
 
 Using the drop-down options by the KQL bar, you can select whether <<det-engine-terminology, alerts>>, other raw events, or both are displayed in the Timeline.
 
+[discrete]
+[[timeline-inspect-events-alerts]]
+== Inspect event or alert
```
Comment (jmikell821) [inline docs/events/timeline-ui-overview.asciidoc:37]:
```suggestion
== Inspect an event or alert
```

**Review by jmikell821 — COMMENTED** [review]
Slight nit, then good to go - thanks!

**Pull request merged**